app/views/shared/results/_search_results.html.haml

```diff
@@ -18,7 +18,7 @@
         with dates
       - if provider_facets.present?
         from
-        = "#{ 'over' if provider_facets.count >= 2000 }"
+        = "#{ 'over' if provider_facets.count >= 240 }"
```

Comment:
@markbreedlove I like the idea of having a nice, even number like 300 instead of 240.